### ISSUE TITLE
Event stream should have headline the document name title (not the path)

### ISSUE THREAD

**Issue by superkai** [HIIT/dime-server#53]
The path is not appropriate or handy title for event entry.The highlight title should be the document title and potentially associated tags, then paths, logger, event type should follow.

**Comment by superkai** [comment]
This is from a user perspective. System perspective might be different. But the UI is for users..

**Comment by jmakoske** [comment]
I agree. But extracting the title e.g. from a PDF document is not trivial. (Assuming it is not in the metadata title field, as is very often the case.)

**Comment by jmakoske** [comment]
For web pages it is of course easy. The title is actually there already, the UI should just show it.

**Comment by MarcoFiletti** [comment]
I just added this since very soon PeyeDF and JustUsed will be able to automatically download metadata for some scientific articles, which increases the number of documents which have a title

**Comment by jmakoske** [comment]
Great!  Another option is to try to extract the title, for example, based on the heuristic that it is the text with the biggest font size on the top half of the first page. For example https://www.docear.org/software/add-ons/docears-pdf-inspector/ seems to manage to do this rather well.

**Comment by MarcoFiletti** [comment]
Good idea! The new commits now try to do this also
